deploy/index.md

```diff
@@ -41,15 +41,34 @@ A CDN deployment of Protomaps has three parts:
 
 ## Next steps
 
-* [Deploy PMTiles on Cloudflare.](/deploy/cloudflare)
-* [Deploy PMTiles on AWS.](/deploy/aws)
-* [Deploy PMTiles with the Caddy web server.](/deploy/server)
+* [Deploy PMTiles on Cloudflare](/deploy/cloudflare)
+* [Deploy PMTiles on AWS](/deploy/aws)
+* [Deploy PMTiles with the Caddy web server](/deploy/server)
 
-## Comparison
+## Deployment Comparison Chart
+
+_Cloudflare is recommended for budget minded beginners. AWS is recommended for advanced developers who require a faster map and/or can offset increased marginal cost with monetization._
 
 | feature | static pmtiles | Cloudflare |  AWS | [Caddy](/deploy/server#caddy) | `pmtiles serve` |
-| - | - | :-| -: | - | - | 
-| Z/X/Y compatible | | ✅ | ✅ |✅ | ✅ |
-|  edge caching | |  ✅ |  ✅ |   With CDN | With CDN |
-|  SSL |  ✅ |  ✅ |  ✅ |  ✅ | With reverse proxy |
-| scale to zero |  ✅ |  5 USD |  ✅ | | |
+| - | :-: | :-:| :-: | :-: | - | 
+| Z/X/Y compatible | ✖️    | ✅        | ✅    | ✅       | ✅                  |
+| Edge caching     | ✖️    |  ✅       |  ✅   | With CDN | With CDN           |
+| SSL              |  ✅  |  ✅       |  ✅   |  ✅       | With reverse proxy |
+| Scale to zero    | ✅   |  $5 (USD) |  ✅   | ✖️        |  ✖️                 |
+| Setup effort     | 😓   | 😓        | 😓    | 😓😓     | 😓😓😓               |
+| Latency          | 🚀   |  🚀       |  🚀🚀🚀 | 🚀🚀     | 🚀🚀🚀              |
```

Comment:
The last two columns are more like `it depends` for latency because they only prescribe a decoding method, not what network it comes from (it could just as well be AWS Cloudfront or Cloudflare)